### ISSUE TITLE
Output of `openvpn-gui --help` is messy and inconsistent

### ISSUE THREAD

**Issue by EagleErwin** [OpenVPN/openvpn-gui#221]
When I was checking if some translations fit in the HMI, I noticed the output of `openvpn-gui.exe --help` (see left image below). The line breaks cause the text to be mixed with the actual command line arguments, some of the semicolons are not aligned and some 1/0 options are explained for both cases, some only for the '1' option.

The first thing I wandered, is why a `MessageBoxEx` is used to show this output, and not stdout. I think the only situation that you are calling `openvpn-gui.exe --help` is when you are already in command line mode so you will see the output when it is on stdout.

If it is desired that the output stays a MessageBoxEx (which seems to have a fixed width) I think the text must be adapted so it fits better in the MessageBox (see right image below). If you want, I can provide a PR for the changes I did to create the screenshot, but eventually it requires changes in the resource files of all languages.

![help](https://user-images.githubusercontent.com/2418451/36446594-18eb8cf6-1682-11e8-8d6d-fff5fcef0033.png)

**Comment by selvanair** [comment]
We love patches/PRs. So please do.
And yes, the --help o/p is crappy. Personally I wouldn't care whether its in a message box, on the cmd line or in a custom window. All are pretty user-unfriendly on Windows.

**Comment by EagleErwin** [comment]
I have experimented with outputting the information on the cmd line, but I found out that TUI and GUI are completely different worlds in win32 and it is impossible to merge them in a sensible way.

I have created a PR for this issue that fixes the layout in the existing help-dialog for some languages. I might want to create a custom window some time, but for now, this seems the most reasonable option for me.
